src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -452,23 +461,37 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        // The instructor fragment is populated by calling the corresponding function along with the arguments.
```

Comment:
This comment doesn't add any value as it simply describes what is already obvious from the code. Rather, it should explain the purpose of the variable `instructorFragment ` (i.e. what it represents).